sphinxodoo/ext/autodoc.py

```diff
@@ -29,8 +29,12 @@ def load_odoo_modules(addons):
             except ImportError:
                 # skip non module directories
                 continue
+            if use_odoo_namespace:
+                namespace = 'odoo'
+            else:
+                namespace = 'openerp'
             mod = imp.load_module(
-                'openerp.addons.%s' % module_name, f, path, descr)
+                '%s.addons.%s' % (namespace, module_name), f, path, descr)
```

Comment:
@sylvain-garancher but if you feel this is better, I certainly won't fight over it and amend the code.